tests/e2e/flowtest/test_base_operation_node_drain.py

```diff
@@ -20,7 +20,7 @@
 
 
 @ignore_leftovers
-@flowtests
+@tier4b
```

Comment:
@bernerhat Can we skip adding tier for this test for now? It is a bit complicated and has been a while since this test has been executed, needs to be stabilized else it will break the entire cluster.

@PrasadDesala